packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -55,6 +55,25 @@ import {
 	type IBlobManagerLoadInfo,
 } from "./blobManagerSnapSum.js";
 
+/**
+ * The BlobManager produces BlobHandles during operation. In older versions of Fluid, these
+ * handles were assumed to only exist if their respective blob was attached and available to
+ * remote clients - these handles are NOT annotated with metadata. Now, the handle may exist
+ * prior to remote availability while the source client completes the upload/attach of the
+ * blob - these handles ARE annotated with metadata.
+ */
+interface IBlobManagerHandleMetadata {
+	readonly type: "blob";
```

Comment:
should this specify if it is created deferred vs not?

Reply:
Seems reasonable - assuming we get rid of the current flow in the long term the distinction becomes uninteresting, but might as well distinguish for now.  Will take a look.

I'll probably do the inverse and have it declare if it is blob-attached, so that once the current flow goes away it'll let us end with less-decorated handles.

Reply:
Done in latest